{"name": "Test login with incorrect login credentials with parameters -- @1.2 Credentials", "status": "broken", "statusDetails": {"message": "AttributeError: 'Context' object has no attribute 'driver'\n", "trace": "  File \"/home/oleg/PycharmProjects/bdd/.venv/lib/python3.10/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/oleg/PycharmProjects/bdd/.venv/lib/python3.10/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/login_steps_with_parameters.py\", line 13, in step_impl\n    SeleniumHelper(context.driver).open_page(LOGIN_URL)\n  File \"/home/oleg/PycharmProjects/bdd/.venv/lib/python3.10/site-packages/behave/runner.py\", line 321, in __getattr__\n    raise AttributeError(msg)\n"}, "steps": [{"name": "When user input wrong \"fisher\" and password \"toxic\"", "status": "broken", "statusDetails": {"message": "AttributeError: 'Context' object has no attribute 'driver'\n", "trace": "  File \"/home/oleg/PycharmProjects/bdd/.venv/lib/python3.10/site-packages/behave/model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"/home/oleg/PycharmProjects/bdd/.venv/lib/python3.10/site-packages/behave/matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features/steps/login_steps_with_parameters.py\", line 13, in step_impl\n    SeleniumHelper(context.driver).open_page(LOGIN_URL)\n  File \"/home/oleg/PycharmProjects/bdd/.venv/lib/python3.10/site-packages/behave/runner.py\", line 321, in __getattr__\n    raise AttributeError(msg)\n"}, "attachments": [{"name": ".text", "source": "70decc3d-dd12-4a45-9d44-83c191366529-attachment.txt", "type": "text/plain"}], "start": 1727621663084, "stop": 1727621663085}, {"name": "Then error message will come with parameters", "status": "skipped", "start": 1727621663085, "stop": 1727621663085}], "parameters": [{"name": "login_id", "value": "fisher"}, {"name": "password", "value": "toxic"}], "start": 1727621663083, "stop": 1727621663085, "uuid": "0a5d0cc9-30d4-443f-b289-a685a683f833", "historyId": "8e286c1660fc5ac8453b89aa61d0d1dd", "fullName": "Facebook login functionality with parameters: Test login with incorrect login credentials with parameters", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Facebook login functionality with parameters"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}